src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -220,28 +214,44 @@ public WorkspaceSelector(@Nullable MCreatorApplication application, WorkspaceOpe
 		});
 		southcenter.add(prefs);
 
-		add("Center",
-				PanelUtils.centerAndSouthElement(PanelUtils.northAndCenterElement(logoPanel, actions), southcenter));
+		JPanel southcenterleft = new JPanel(new FlowLayout(FlowLayout.LEFT));
+
+		JLabel version = L10N.label("dialog.workspace_selector.version", Launcher.version.getMajorString());

```

Comment:
Maybe add this below the `logoPanel` either next to `socialButtons` or above them?